examples/next/pages/_app.page.tsx

```diff
@@ -0,0 +1,9 @@
+import { AmplifyProvider } from '@aws-amplify/ui-react';
+
+export default function ExampleApp({ Component, pageProps }) {
+  return (
+    <AmplifyProvider>
```

Comment:
This doesn't seem to actually work.

Run `yarn next-example dev` & checkout http://localhost:3000/ui/components/authenticator/sign-up-with-username